setup.py

```diff
@@ -8,6 +8,8 @@
 
 deps = [
     'marionette-client == 0.9',
+    'mozinstall == 1.11',
+    'mozlog == 2.10',
```

Comment:
Now I'm not sure. If we end up with multiple versions of mozlog that sounds bad, but maybe that's dealt with automatically.